### ISSUE TITLE
Searching with lists 

### ISSUE THREAD

**Issue by debhogan-208** [greenelab/adage-frontend#178]
On two browsers (Chrome and Safari), searching with a  list of 14 genes gave two to four identical returns (the links suggested they were the same). They weren't always contiguous but they all highlighted when you clicked one. 

<img width="1367" alt="Screenshot 2020-05-03 09 09 50" src="https://user-images.githubusercontent.com/63237521/80915154-52729e00-8d1e-11ea-8b5a-594beb2a5eaf.png">
<img width="1400" alt="Screenshot 2020-05-03 09 11 01" src="https://user-images.githubusercontent.com/63237521/80915156-530b3480-8d1e-11ea-84a2-ecc4b6cd391e.png">

**Comment by vincerubinetti** [comment]
This is behaving as intended. Searches on the backend might return the same set of results for two similar search strings. Then, if you click to select one gene, if that same gene shows up in a different search result, the checkmark status will be the same, because it's the same gene. If that gene is selected, it should show a checkmark to indicate to the user that it's selected, and that clicking it will unselect it.

In other words, what genes are selected is global to the whole app, not local to each search string in the multi search.

If you believe that two different search strings should be returning significantly different results, we'll have to talk to @dongbohu about tweaking the search weights and etc.

**Comment by dongbohu** [comment]
When searching a list of genes, it may make more sense to do exactly match (case-insensitive) on each one (instead of fuzzy match).

**Comment by cgreene** [comment]
I like @dongbohu's idea. If the match scores are directly comparable between search terms, you could also only display a potential match as an option in the cases where it had the highest score. However, this would require that scores be comparable b/w queries.

**Comment by vincerubinetti** [comment]
I implemented this a while ago and haven't received confirmation that it's okay. But it stands to reason (imo) that if you're searching for a list of genes, you'll want more exact matches, so the fix makes sense to me. Closing for now.
